apps/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from . import views
+
+app_name = "api"
+
+urlpatterns = [
+    path("experiments/", views.get_experiments, name="list-experiments"),
+    path("participants/experiments/data", views.update_participant_data, name="update-participant-data"),
```

Comment:
what about making this more RESTy?
```
POST /participants/<id>/<experiment-id>/
    body: {"name": "John"}

OR

POST /participants/<id>/
    body: {"<experiment-id>": {"name": "John"}}
    OR 
    body: {"data": [{"experiment_id": "<experiment-id>", "name": "John"}]}

```

Reply:
I like the second one. That way we can bulk update participant data for various experiments